### PR TITLE
Release Network-Operator v0.4.0 (#137)

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ optionally deploy Node Feature Discovery in the cluster. This is enabled via `nf
 ### SR-IOV Network Operator
 Nvidia Network Operator can operate in unison with SR-IOV Network Operator
 to enable SR-IOV workloads in a Kubernetes cluster. We provide a helm chart to be used to optionally
-deploy [SR-IOV Network Operator](https://github.com/k8snetworkplumbingwg/sriov-network-operator) in the cluster.
+deploy [SR-IOV Network Operator](https://github.com/openshift/sriov-network-operator) in the cluster.
```
Comment (moshe010) [inline README.md:34]: this undo me commit. you rebase it wrong

**Review by moshe010 — COMMENTED** [review]
update the commit msg to release 0.4.0

File: README.md (hunk 1 above)
Comment (e0ne) [reply README.md:34]: We need to change it in master too https://github.com/Mellanox/network-operator/blame/master/deployment/network-operator/README.md#L34

File: README.md [hunk 2]
```diff
@@ -203,7 +224,7 @@ the file: `values.yaml`
 Specifies components to deploy in order to facilitate a secondary network in Kubernetes. It consists of the following optionally deployed components:
   - [Multus-CNI](https://github.com/k8snetworkplumbingwg/multus-cni): Delegate CNI plugin to support secondary networks in Kubernetes
   - CNI plugins: Currently only [containernetworking-plugins](https://github.com/containernetworking/plugins) is supported
-  - IPAM CNI: Currently only [Whereabout IPAM CNI](https://github.com/k8snetworkplumbingwg/whereabouts) is supported
+  - IPAM CNI: Currently only [Whereabout IPAM CNI](httpso://github.com/k8snetworkplumbingwg/whereabouts) is supported
```
Comment (moshe010) [inline README.md:227]: typeo httpso

File: README.md (hunk 1 above)
Comment (moshe010) [reply README.md]: yes please do it on master as well

**Review by moshe010 — APPROVED** [review]

**Pull request merged**